File: Week2/activity/classify.c
#include <stdio.h>
int main(){
	int ls[] = {'a','b','c'};
	int length = (int)( sizeof(ls) / sizeof(ls[0]));
	if (length % 2 == 0)
		printf("even number of elements in the list\n");
	else
		printf("odd number of elements in the list\n");
	return 0;
}

File: Week2/activity/classify..pl
even:-
    write('even elements in the list\n').
odd:-
    write('odd elements in the list\n').
classify([H|T]):-
    length(T,M),
    M>=0 ->
    (
        N is mod(M+1,2),
        N=:=0 -> even;odd
    ).
:- initialization(main).
main :- 
classify(['a','b','c','d','e']),
classify(['a','b','c','d','e','f']).



